src/webview/html/messageAsHtml.js

```diff
@@ -118,7 +129,18 @@ $!${messageDiv(id, 'message-full', flags)}
   </div>
   <div class="content">
     $!${messageSubheader({ fromName, timestamp, twentyFourHourTime })}
-    $!${messageBody({ content, flags, id, isOutbox, ownEmail, reactions, realmEmoji, timeEdited })}
+    <div dir="auto" class="message-content">
```

Comment:
Here it is a bit tricky to add the `dir="auto"` to the `<div class="content">` since the subheader will trick the algorithm. But maybe we can move the subheader out of that `div`.